user-service/src/main/java/uk/gov/gchq/palisade/service/user/exception/ApplicationAsyncExceptionHandler.java

```diff
@@ -33,7 +35,8 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(ApplicationAsyncExceptionHandler.class);
 
     @Override
-    public void handleUncaughtException(final Throwable throwable, final Method method, @Nullable final Object... objects) {
+    public void handleUncaughtException(final Throwable throwable, final Method method, @Nullable final Object[] objects) {
+        requireNonNull(objects);
```

Comment:
Optional null check instead of Object check.

Reply:
Updated